src/ui/components/Spinner.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { memo, useEffect, useRef } from 'react'
+import { Animated, Easing, Platform } from 'react-native'
+import styled from 'styled-components/native'
+
+import { Logo } from 'ui/svg/icons/Logo'
+import { IconInterface } from 'ui/svg/icons/types'
+import { ColorsEnum } from 'ui/theme'
+
+function NotMemoizedSpinner({ size, color = ColorsEnum.GREY_DARK }: IconInterface) {
+  const animatedValue = useRef(new Animated.Value(0)).current
+  const spin = animatedValue.interpolate({
+    inputRange: [0, 1],
+    outputRange: ['0deg', '360deg'],
+  })
+
+  useEffect(() => {
+    Animated.loop(
+      Animated.timing(animatedValue, {
+        toValue: 1,
+        duration: 3000,
+        easing: Easing.linear,
+        useNativeDriver: ['ios', 'android'].includes(Platform.OS),
```

Comment:
```suggestion
const USE_NATIVE_DRIVER = Platform.select({ default: false, iOS: true, Android: true })
```

à définir au dessus :) mais sinon c'est bien